apps/users/views.py

```diff
@@ -231,6 +232,78 @@ def edit(request):
 
 @write
 @login_required
+def t_shirt(request):
+    if not waffle.switch_is_active('t-shirt-orders'):
+        raise http.Http404()
+
+    MIN_PERSONA_ADU = 10000
+
+    user = UserProfile.objects.get(pk=request.user.id)
```

Comment:
Why not simply use `request.user`? And `request.user.tshirtorder_set.get()` for the shirt order?